src/main/java/sboot/example/util/ReviewDtoMapper.java

```diff
@@ -0,0 +1,30 @@
+package sboot.example.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import sboot.example.dto.ReviewDto;
+
+@RequiredArgsConstructor
+public class ReviewDtoMapper {
+    private final CsvReaderUtil readerUtil;
+
+    public List<ReviewDto> getReviewDto() {
```

Comment:
mapper it's not a repository, you cannot just get something from there. This class should convert to dto or from dto